### DISCUSSION TITLE
买了2年专业版，登不上了

### DISCUSSION THREAD

**Discussion by sam-unc** [getlantern/lantern-forum-cn#783] [Q&A]
@Mike777Khoo
8月3日购买的，登录不上了。麻烦帮看看。是不是续期付款时没有登录啊，邮箱：zhf7912@hotmail.com
![Uploading 微信图片_20240813211252.jpg
![微信图片_20240813211257](https://github.com/user-attachments/assets/8a89af63-5c2d-4c2a-985a-e15e8c66e336)
![Uploading 微信图片_20240813211257.jpg…]()
…]()

**Comment by sam-unc** [comment]
![微信图片_20240813211252](https://github.com/user-attachments/assets/4965d016-a5cc-44f8-ab5d-ae80d6ba5162)
![微信图片_20240813211257](https://github.com/user-attachments/assets/6f841dd7-4438-4663-95f3-85dd532a326e)

**Comment by MaxMKMax** [comment]
@sam-unc 您好. 好的, 我已经帮您提交问题上去了.

**Comment by MaxMKMax** [comment]
@sam-unc 您好. 我已经发激活码到您的邮箱了. 请查看. 谢谢您耐心的等待!
